packages/neuron-wallet/src/block-sync-renderer/sync/queue.ts

```diff
@@ -40,13 +40,16 @@ export default class Queue {
 
   private assetAccountInfo: AssetAccountInfo | undefined
 
-  constructor(url: string, lockHashes: string[], multiSignBlake160s: string[], startBlockNumber: bigint) {
+  private anyoneCanPayLockHashes: string[]
+
+  constructor(url: string, lockHashes: string[], anyoneCanPayLockHashes: string[], multiSignBlake160s: string[], startBlockNumber: bigint) {
```

Comment:
There're too many params, consider using an object instead of a param list.

Reply:
Seems no need, any obvious benefits ?

Reply:
> Seems no need, any obvious benefits ?

Not necessary, just a best practice that the params in the list should be up to 3